geo-location.html

```diff
@@ -67,13 +68,22 @@
         },
 
         /**
+         * If true, the element won't be active at all
+         */
+        idle: {
+          type: Boolean,
+          value: false,
+          observer: 'fetch'
```

Comment:
changing `idle` to `true` stops the watch if that's active and does nothing. Changing it to `false` starts the watch (or fetch) again.